docs/features/blocks/contentblocks/googlemap.rst

```diff
@@ -0,0 +1,33 @@
+Google Map Block
+================
+
+Allows the user to add a Google Map location and display the location on the map. 
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **Search Query** - Address or search term used to find your location on the map.
+
+* **Map Title** - Map title for screen readers, ex: "Map to Simple Sweet Desserts"
+
+* **Google Place ID** - Place IDs uniquely identify a place in the Google Places database and on Google Maps
+
+.. note::
+    Google Place IDs require a Google API Key. Read more about `Google Place IDs <https://developers.google.com/places/place-id>`_.
+
+* **Map Zoom Level** - Requires API key to use zoom. ex: 1: World, 5: Landmass/continent, 10: City, 15: Streets, 20: Buildings
+
+* **Advanced Settings** - Add custom CSS classes or a CSS ID to style the block with your custom CSS
+
+.. figure:: img/googlemap_settings.png
+    :alt: Google Map block with its Settings
```

Comment:
Can you use a real address here, and then show the output map as well? (You can use our address 55 Public Sq)

Reply:
done